docs/effects.md

```diff
@@ -1,4 +1,20 @@
-### MoveEffect
+# Effects
+An effect can be applied to any `PositionComponent`, there are currently two effects that you can use and that is the MoveEffect and the ScaleEffect.
+
+If you want to create an effect that only runs once, specify the three first parameters Size/Position, Speed and Curve.
+
+## More advanced effects
+Then there are two optional boolean parameters called `isInfinite` and `isAlternating`, by combining them you can get different effects.
+
+If both of them are true the effect will be infinite and when it is done with going through the `curve` it will go back in reverse through the `curve`.
+
+If `inInfinite` is true and `isAlternating` is false the effect will be applied from start to end of the curve and then restart from the start, in infinity.
```

Comment:
There are several typos on these phrases: inInfinite

Reply:
I must have been very tired when writing this, good catch!